src/main/kotlin/parts/wisdom/arcraider/visualization/GridPane.kt

```diff
@@ -0,0 +1,42 @@
+package parts.wisdom.arcraider.visualization
+
+import java.awt.Color
+import java.awt.Graphics
+import javax.swing.JPanel
+
+// When calculating the width / height of each grid pixel, offset it so
+// we don't fill right to the edge of the screen.
+private const val OFFSET = 10
```

Comment:
Could this have a more descriptive name? Maybe it's `PANEL_INSET_PIXELS`?

Reply:
fixed